### PR TITLE
Update the README to include a link to the Polkadot SDK Version Manager

### PR DESCRIPTION
**Pull request by iammasterbrucewayne**
Adds a link to the [Polkadot SDK Version Manager](https://github.com/paritytech/psvm) since this tool is not well known, but very useful for developers using the Polkadot SDK.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -81,3 +81,6 @@ fellowship, this separation, the RFC process
 This repository is the amalgamation of 3 separate repositories that used to make up Polkadot SDK,
 namely Substrate, Polkadot and Cumulus. Read more about the merge and its history
 [here](https://polkadot-public.notion.site/Polkadot-SDK-FAQ-fbc4cecc2c46443fb37b9eeec2f0d85f).
+
+## Other useful resources and tooling
```
Comment (bkchr) [inline README.md:85]:
```suggestion
## Other useful resources and tooling

```

**Review by bkchr — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -81,3 +81,7 @@ fellowship, this separation, the RFC process
...
+
+* A simple tool to manage and update the Polkadot SDK dependencies (https://github.com/paritytech/psvm)
```
Comment (ggwpez) [inline README.md:87]:
```suggestion
* A simple tool to manage and update the Polkadot SDK dependencies (https://github.com/paritytech/psvm).
```

**Review by ggwpez — APPROVED** [review]

**Comment by kianenigma** [comment]
Would have been nicer if this was added in the section under releases section and proper linking was used ;)

**Comment by iammasterbrucewayne** [comment]
@kianenigma will this work? :)

https://github.com/paritytech/polkadot-sdk/pull/4734

**Pull request merged**